app/src/main/java/org/breezyweather/common/ui/activities/AlertActivity.kt

```diff
@@ -151,10 +152,23 @@ class AlertActivity : GeoActivity() {
                 }
             }
         }
-
         val scrollBehavior = generateCollapsedScrollBehavior()
 
-        BreezyWeatherTheme(lightTheme = MainThemeColorProvider.isLightTheme(context, location.value)) {
+        val isLightTheme = MainThemeColorProvider.isLightTheme(context, location.value)
+        BreezyWeatherTheme(lightTheme = isLightTheme) {
+            //re-setting the status bar color once the location is fetched above in the launched effect
+            ThemeManager
+                .getInstance(this)
+                .weatherThemeDelegate
+                .setSystemBarStyle(
+                    context = this,
+                    window = this.window,
+                    statusShader = false,
+                    lightStatus = isLightTheme,
+                    navigationShader = true,
```

Comment:
One more thing, the navigation bar now has a shade while it previously had a solid color. I'm actually not sure what's better. Although it won't matter that much once the screens have edge-to-edge support.

Reply:
Actually, I checked the usages of `Window.setSystemBarStyle()` before and found that `GeoActivity `and `ManagementFragment` were sending the shader as true and event `HomeFragment `call to the `setSystemBarStyle()` has the `navigationShader` as `true`. So I just maintained consistent behavior. 

I ran the app just now to check in my device and here are the screenshots for the `navigationShader` `true` scenario:

![Screenshot_20240502_210025](https://github.com/breezy-weather/breezy-weather/assets/7792665/b8742de8-e030-40eb-b44e-8e9e071e2e1e)


![Screenshot_20240502_210140](https://github.com/breezy-weather/breezy-weather/assets/7792665/3bc1f006-5938-4e54-9047-da3777d106a7)

Let me know if this looks good